library/js/oeUI/insurance/EditPolicyScreenController.js

```diff
@@ -0,0 +1,910 @@
+                        errorNode = errorNode.nextElementSibling ? errorNode.nextElementSibling.querySelector('.select2-selection') : null;
+                    }
+                    if (errorNode) {
+                        errorNode.classList.add('invalid-input');
+                        errorNode.classList.add('error');
+                        errorNode.classList.add('error-border');
+                    }
+                }
+            });
+        }
+    }
+
+    #populateSelectDropdowns() {
+        this.__types.forEach(type => {
+            let select = document.getElementById('insurance-type-' + type);
+            if (select) {
+                select.addEventListener("change", (evt) => {
+                    let selectedId = evt.target.value;
+                    let type = evt.target.dataset.type;
+                    if (!selectedId) {
+                        selectedId = null; // so we can find the blank policy
+                    }
+                    let updatedInsurance = this.__insurancesByType[type].find(insurance => insurance.id == selectedId);
+                    this.#updateSelectedInsuranceObject(updatedInsurance);
+                    this.render();
+                });
+            }
+        });
+    }
+
+    #populateInsuranceInformationForSelectedInsurance(selectedInsurance) {
+        if (selectedInsurance) {
+            try {
+                let type = selectedInsurance.type;
+                let insuranceInfoContainer = document.getElementById('insurance-info-type-' + type);
+                if (insuranceInfoContainer) {
+                    this.clearEvents(); // clear any bound events that we've done here
+                    insuranceInfoContainer.innerHTML = '';
+                    let template = document.getElementById('insurance-edit-template');
+                    if (!template) {
+                        throw new Error("Failed to find insurance edit template");
+                    }
+
+                    let clone = document.importNode(template.content, true);
+                    insuranceInfoContainer.appendChild(clone); // note clone nodes are now empty at this point
+                    this.#populateInsuranceProviderListForNode(insuranceInfoContainer, this.__insuranceProviderList);
+                    // the select2 setup needs to occur after the data elements have been populated.
+                    this.#populateInsuranceModelDataElements(insuranceInfoContainer, selectedInsurance);
+                    this.#setupModelSyncBinding(selectedInsurance);
+                    this.#setupAddressValidation(selectedInsurance);
+                    this.#setupInsuranceSwapButtons(selectedInsurance);
+                }
+            }
+            catch (error) {
+                console.error(error);
+                alert(window.top.xl("An error occurred while rendering the insurance policy.  Please try again or contact your system administrator."));
```

Comment:
only use single spacing in translated constants

Reply:
Fixed.